### PR TITLE
Document how to modify the environment section after tests are finished 

### PR DESCRIPTION
**Pull request by gnikonorov**
As per the title, explain how to modify the environment section in a report after tests are finished.

Change made as per https://github.com/pytest-dev/pytest-html/issues/373#issuecomment-739431537

Closes https://github.com/pytest-dev/pytest-html/issues/373

**NOTE**: This needs to be moved if https://github.com/pytest-dev/pytest-html/pull/402 merged before this PR is in

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -102,13 +102,33 @@ Environment
...
+.. code-block:: python
+
+  import pytest
+
+
+  @pytest.hookimpl(tryfirst=True)
+  def pytest_sessionfinish(session, exitstatus):
+      session.config._metadata["Average Response Time"] = calculate_avg_response_time(
+          session
+      )
```
Comment (BeyondEvil) [inline README.rst:125]:
```suggestion
      session.config._metadata["foo"] = "bar"
```

File: README.rst [hunk 2]
```diff
@@ -102,13 +102,33 @@ Environment
...
+
+Note that in the above example `@pytest.hookimpl(tryfirst=True) <https://docs.pytest.org/en/stable/writing_plugins.html#hook-function-ordering-call-example>`_
+is important, as this ensures your :code:`pytest_sessionfinish` method is run **before** any other plugins ( including :code:`pytest-html` and
```
Comment (BeyondEvil) [inline README.rst:128]: Not sure "ensure" is factually correct here (not sure that it matters either 😆 ), but I think pytest doesn't guarantee it, but they will try their best. :)

**Review by BeyondEvil — APPROVED** [review]
Awesome @gnikonorov !

Thanks for everything you're doing for the plugin! ❤️ 💪

File: README.rst (hunk 2 above)
Comment (gnikonorov) [reply README.rst:128]: sure, I'll update the wording

**Comment by gnikonorov** [comment]
Should be good for another look @BeyondEvil. I moved the change to the read the docs docs

**Review by BeyondEvil — APPROVED** [review]
Nice!

**Comment by gnikonorov** [comment]
Thanks @BeyondEvil! I just merged in master. If you have no issues with this, can you please perform the merge?

**Pull request merged**